script/BaseMigrationV2.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import "foundry-deployment-kit/BaseMigration.s.sol";
+
+abstract contract BaseMigrationV2 is BaseMigration {
```

Comment:
please use `BaseMigrationExtended` instead of V2

Reply:
will resolve later